File: project_bundles_v2/python/app/main.py
def main():
    values = [12, 18, 21, 9, 31]
    total = sum(values)
    avg = total / len(values)
    print(f'Total: {total}')
    print(f'Average: {avg:.2f}')

if __name__ == '__main__':
    main()

File: project_bundles_v2/java/app/main.java
public class Main {
    public static void main(String[] args) {
        int[] values = {12, 18, 21, 9, 31};
        int total = 0;
        for (int value : values) total += value;
        System.out.println("Total: " + total);
        System.out.println("Average: " + (total / (double) values.length));
    }
}
